tools/CMakeLists.txt

```diff
@@ -182,11 +182,17 @@ foreach (deploytgt drconfig drrun drinject)
     target_link_libraries(${deploytgt} drinjectlib)
   endif ()
   if (LINUX) # No checks for MacOS
-    get_target_property(check_path ${deploytgt} LOCATION)
-    add_custom_command(TARGET ${deploytgt}
-      POST_BUILD
+    set(locvar_rel "")
+    if ("${CMAKE_VERSION}" VERSION_EQUAL "3.10" OR
+        "${CMAKE_VERSION}" VERSION_GREATER "3.10")
+      set(locvar_rel "../")
```

Comment:
This is being used multiple times: I think this should be in a shared function so we have the comment explaining what it is in one place.  Maybe the function could include the file(GENERATE if that's easy to do.

Reply:
The relative path depends on where the caller is located though. So putting it in a function (though we only need '../') makes it less flexible, unlessI'd add some weird parameter like "levels_needed".

It's only need in tools/CMakeLists.txt and core/CMakeLists.txt.

you ok with leaving it as is for now?

Reply:
If we use absolute paths we can eliminate all of this cmake-version-specific code.  We could then have a function that invokes file(GENERATE if that seems useful to avoid duplication, or not if we think the code is small enough once this "../" adjustment disappears.

Reply:
Done